cmd/zed/lake/index/add.go

```diff
@@ -0,0 +1,119 @@
+package index
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+
+	zedlake "github.com/brimdata/zed/cmd/zed/lake"
+	"github.com/brimdata/zed/pkg/charm"
+	"github.com/brimdata/zed/pkg/rlimit"
+	"github.com/segmentio/ksuid"
+)
+
+var Add = &charm.Spec{
+	Name:  "add",
+	Usage: "add [options] [ -rule ruleid ] tag [ ...tag ]",
```

Comment:
Follow the lead in `cmd/zed/lake/commit/command.go` and `cmd/zed/lake/squash/command.go`.
```suggestion
	Usage: "add [options] [ -rule ruleid ] tag [tag ...]",
```